.github/workflows/publish-module.yml

```diff
@@ -70,7 +70,7 @@ jobs:
           bicep --version
 
       - name: Publish module
-        run: bicep publish "modules/${{ steps.parse-tag.outputs.module_path }}/main.json" --target "br:${{ secrets.PUBLISH_REGISTRY_SERVER }}/public/bicep/${{ steps.parse-tag.outputs.module_path }}:${{ steps.parse-tag.outputs.version }}" --documentationUri "${{ steps.parse-tag.outputs.documentation_uri }}" --force
+        run: bicep publish "modules/${{ steps.parse-tag.outputs.module_path }}/main.json" --target "br:${{ secrets.PUBLISH_REGISTRY_SERVER }}/public/bicep/${{ steps.parse-tag.outputs.module_path }}:${{ steps.parse-tag.outputs.version }}" --documentationUri "${{ steps.parse-tag.outputs.documentation_uri }}" --with-source --force
```

Comment:
This pipeline is not used by AVM, but only by the `modules` folder. 
If we agreed to proceed separately we can revert this change

Reply:
This will still work for BRM modules once they pass their tests to be published so can be left here.

Reply:
Please don't self-resolve comments. Makes re-reviews a lot harder 😉

To the topic at hand: Fair point, if they pass, they pass.

Reply:
Fair, this can stay here though, there's no harm to add this the BRM modules publishing flow. They issues with those modules is not related to the publishing part, so this gets it in for when/if they are fixed for diagnostic settings etc.

Reply:
Sounds good. As per latest discussions, this workflow will likely never be run anyway.